demo/settings_USA.yaml

```diff
@@ -26,9 +26,28 @@ search:
     - MONSTER
 
   # Region that we are searching for jobs within:
-  province_or_state: "Texas"  # NOTE: this is generally 2 characters long.
-  city: "Richardson"  # NOTE: this is the full city / town name.
+  # NOTE: this is generally 2 characters long, but can be 'Null' (no quotes)
+  # to search by country, or can be some value while city is 'Null' to search
+  # within the province or state.
+  province_or_state: "Texas"
+  # NOTE: this is the full city / town name, but can be 'Null', with
+  # province_or_state set to 'Null' to search country-wide.
+  city: "Richardson"
```

Comment:
The way the schema has been implemented, we can simply not provide `province_or_state` / `city` entirely and they will be set to the default value in the schema, which should be None. 

i.e. user shouldn't be setting this to a `Null` string